content/dev/autograders.md

```diff
@@ -2,4 +2,74 @@
 
 # Autograders
 
-stub
\ No newline at end of file
+PrairieLearn supports [external autograders](https://prairielearn.readthedocs.io/en/latest/externalGrading/) for assessment of students' submissions that is highly customizable. A common use case is to assess student-submitted code against tests and give a grade and feedback based on the test results.
+
+If you are working in [Python](https://prairielearn.readthedocs.io/en/latest/python-grader/), [C or C++](https://prairielearn.readthedocs.io/en/latest/c-grader/), or [Java](https://prairielearn.readthedocs.io/en/latest/java-grader/), you will likely want to use the provided autograders.
+
+If you have your own "boutique" language, someone in the PrairieLearn community may already have developed an autograder they can share with you. ([Haskell anyone](https://github.com/steven-wolfman/haskell-prairielearn)?) You can also take an existing autograder and adapt it to your needs.
+
+## Cool Questions 4
+
+What can you do with your autograder? Let's try out some examples that show what's possible.
+
```

Comment:
I have added (https://github.com/PrairieLearn/pl-prairielearn101/pull/3) a couple of questions using the default Java autograder. Does it make sense to start with an "official" autograder before the custom ones? I personally like the Java one as a starting example since it uses JUnit, which many people will be already familiar with.